Frameworks/Account/Feedly/FeedlyAccountDelegate.swift

```diff
@@ -133,6 +143,110 @@ final class FeedlyAccountDelegate: AccountDelegate {
 	func accountDidInitialize(_ account: Account) {
 //		accountMetadata = account.metadata
 		credentials = try? account.retrieveCredentials(type: .oauthAccessToken)
+		
+		caller.getCollections { result in
+			switch result {
+			case .success(let collections):
+				
+				let allFolders = account.folders ?? Set()
+				let allFeeds = account.flattenedFeeds()
+				
+				for collection in collections {
+					let correspondingFolder: Folder? = {
+						for folder in allFolders {
+							if folder.name == collection.label {
+								return folder
+							}
+						}
+						return nil
+					}()
+					
+					let targetFolder: Folder? = {
+						if let folder = correspondingFolder {
+							return folder
+						} else {
+							return account.ensureFolder(with: collection.label)
+						}
+					}()
+					
+					guard let folder = targetFolder else {
+						continue
+					}
+					
+					for feed in collection.feeds {
+						
+						let existingFeed: Feed? = {
+							for nnwFeed in allFeeds where nnwFeed.url == feed.feedId {
+								return nnwFeed
+							}
+							return nil
+						}()
+						
+						let targetFeed: Feed = {
+							if let feed = existingFeed {
+								return feed
+							} else {
+								let url = feed.id
+								let metadata = FeedMetadata(feedID: url)
+								metadata.name = feed.title
+								let feed = Feed(account: account, url: url, metadata: metadata)
+								return feed
+							}
+						}()
+						if !folder.has(targetFeed) {
+							folder.addFeed(targetFeed)
+						}
+					}
+					
+					self.caller.getStream(for: collection) { result in
+						switch result {
+						case .success(let stream):
+							
+							let parsed = stream.items.compactMap { entry -> ParsedItem? in
+								guard let origin = entry.origin else {
+									return nil
+								}
+								let parsed = ParsedItem(syncServiceID: entry.id,
+														uniqueID: entry.id,
+														feedURL: origin.streamId,
+														url: nil,
+														externalURL: origin.htmlUrl,
+														title: entry.title,
+														contentHTML: entry.content?.content,
+														contentText: nil,
+														summary: nil,
+														imageURL: nil,
+														bannerImageURL: nil,
+														datePublished: entry.published,
+														dateModified: entry.updated,
+														authors: nil,
+														tags: nil,
+														attachments: nil)
+								return parsed
+							}
+							
+							for feed in account.flattenedFeeds() {
+								let articles = parsed.filter { item -> Bool in
+									return item.feedURL == feed.feedID
+								}
+								guard !articles.isEmpty else {
+									continue
+								}
+								account.update(feed, parsedItems: Set(articles)) {
+									print("done")
+								}
+							}
+							
+						case .failure(let error):
+							print(error)
+						}
+					}
+				}
+				
+			case .failure(let error):
+				print(error)
+			}
+		}
```

Comment:
Just to be super clear: I know this is a dumpster fire and I've no deliberate intention of this landing in master. This was for me to experiment with getting content from Feedly into NNW and get some feedback as to API (mis-)use or whatever.